soccer/gameplay/role_assignment.py

```diff
@@ -18,6 +18,9 @@ def __init__(self):
         self.priority = 0
         self.require_kicking = False
 
+        # multiply this by the distance between two points to get the cost
+        self.PositionCostMultiplier = 1.0
```

Comment:
Go ahead and rename this to lowercase with underscores so it matches the rest of the instance variables.  The capital letter style is more for global constants.